tests/foreman/cli/test_contentview.py

```diff
@@ -885,9 +884,6 @@ def test_negative_add_same_puppet_repo_twice(self):
 
         @assert: User cannot add puppet modules multiple times to the content
         view
-
-        @bz: 1222118
-
         """
```

Comment:
maybe the entire BZ link is best. That will allow easy copy and paste on the browser in order to visit the bug.